config/jobs/kubernetes/sig-node/node-kubelet.yaml

```diff
@@ -200,8 +200,8 @@ periodics:
           cpu: 4
           memory: 6Gi
       env:
-        - name: IGNITION_INJECT_GCE_SSH_PUBLIC_KEY_FILE
-          value: "1"
+        - name: GOPATH
```

Comment:
IGNITION_... is only needed on jobs that setup the prow user afaik, but only one of these jobs had it configured and both failed in the same way.